exec/cnex/format.h

```diff
@@ -0,0 +1,23 @@
+#ifndef FORMAT_H
+#define FORMAT_H
+#include <stdint.h>
+
+#include "exec.h"
+#include "nstring.h"
```

Comment:
Same with `nstring.h`, because you only ever use pointers to `TString` in this file.

Reply:
Removed include.